From 727171bdc7960b26eff72f0fa990205a3d125c2f Mon Sep 17 00:00:00 2001
From: Gatien Chevallier <gatien.chevallier@foss.st.com>
Date: Tue, 28 May 2024 16:02:21 +0200
Subject: [PATCH 0914/1695] plat-stm32mp1: default enable CFG_DRIVERS_FIREWALL

Default enable the CFG_DRIVERS_FIREWALL switch that is used to enable
the support of the firewall framework.

On this platform, only the ETZPC is a firewall controller for now.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 3332ee274c7923f0b5812a574062c3e01333ca2f plat-stm32mp1: default enable CFG_DRIVERS_FIREWALL]
Change-Id: I30dde583b28377e9ed95c1e1a69e6baace0c9cfd
Signed-off-by: Gatien Chevallier <gatien.chevallier@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/383375
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
Tested-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 core/arch/arm/plat-stm32mp1/conf.mk | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/core/arch/arm/plat-stm32mp1/conf.mk b/core/arch/arm/plat-stm32mp1/conf.mk
index 76b55b9f0..76f3fa4c6 100644
--- a/core/arch/arm/plat-stm32mp1/conf.mk
+++ b/core/arch/arm/plat-stm32mp1/conf.mk
@@ -333,8 +333,9 @@ ifneq ($(CFG_WITH_SOFTWARE_PRNG),y)
 $(call force,CFG_STM32_RNG,y,Required by HW RNG when CFG_WITH_SOFTWARE_PRNG=n)
 endif
 
+CFG_DRIVERS_FIREWALL ?= y
 ifeq ($(CFG_STM32_ETZPC),y)
-$(call force,CFG_STM32_FIREWALL,y)
+$(call force,CFG_DRIVERS_FIREWALL,y)
 endif
 
 ifeq ($(CFG_STPMIC1),y)
-- 
2.47.2

